processing/src/main/java/io/druid/segment/incremental/OnheapIncrementalIndex.java

```diff
@@ -139,6 +145,7 @@ protected DimDim makeDimDim(String dimension, Object lock)
   protected Integer addToFacts(
       AggregatorFactory[] metrics,
       boolean deserializeComplexMetrics,
+      boolean reportParseExceptions,
```

Comment:
this flag is already passed in constr, do we really need to pass it again here ?